ui/components/institutional/jwt-dropdown/jwt-dropdown.scss

```diff
@@ -0,0 +1,13 @@
+.custody-search-jwt__select {
+  height: 54px;
+  width: 100%;
+  display: flex;
+  align-items: center;
+
+  &-title {
+    @include Paragraph;
+
+    color: var(--scorpion);
```

Comment:
We no longer use this color in the extension; could it be `var(--color-error-default)`?